modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/SortedIndex.java

```diff
@@ -90,6 +90,8 @@ void destroy(int partitionId, WriteBatch writeBatch) throws RocksDBException {
 
     @Override
     public void close() {
+        storages.values().forEach(RocksDbSortedIndexStorage::close);
+
         indexCf.handle().close();
```

Comment:
Why don't you use "closeAll" here as well?

Reply:
Fix it.